src/pylint_odoo/checkers/odoo_addons.py

```diff
@@ -733,8 +740,16 @@ def _get_assignation_nodes(self, node):
         "translation-field",
         "translation-positional-used",
         "translation-required",
+        "missing-return",
     )
     def visit_call(self, node):
+        # compute method names must be stored to avoid false positives if super is called inside them
+        if self.linter.is_message_enabled("missing-return"):
+            if hasattr(node, "keywords"):
+                for keyword in node.keywords:
```

Comment:
```suggestion
                for keyword in getattr(node, "keywords", []):
```

That way, you don't need the above conditional, so you may avoid an extra indentation level.

Or in fault, combine the two conditionals above

Reply:
That's nice, I made the change and also added a break once the `compute` attribute is found since there is no use in looking for a second one AFAIK.